src/Sentry/SentryOptions.cs

```diff
@@ -795,6 +797,7 @@ public IList<SubstringOrRegexPattern> FailedRequestTargets
     ///   </item>
     /// </list>
     /// </summary>
+    [Obsolete("Use TracesSampleRate or TracesSampler instead")]
```

Comment:
Does it make sense to be more descriptive here?

Reply:
I played around with the wording on this. There's quite a bit of detail though - I think people will be able to work it out from the docs on the `TracesSampelRate` and `TracesSampler` options.